src/ui/pages/sett/zcl_abapgit_gui_page_sett_locl.clas.abap

```diff
@@ -279,6 +280,11 @@ CLASS zcl_abapgit_gui_page_sett_locl IMPLEMENTATION.
       iv_readonly = boolc( li_package->are_changes_recorded_in_tr_req( ) = abap_false )
       iv_label    = 'BETA: Enable abapGit flow for this repository (requires transported packages)' ).
 
+    ro_form->textarea(
+      iv_name        = c_id-exclude_remote_paths
+      iv_label       = 'Exclude Remote Paths'
```

Comment:
The field label is a bit confusing/inconsistent with [`Ignore Files`](https://docs.abapgit.org/user-guide/repo-settings/dot-abapgit.html#ignore-files). "Ignore" vs "Exclude". How about "Ignore Files (Locally)"?

Reply:
Yeah, the naming is not idea. But it is not really an ignore ... what about "Exclude paths from synronization" ?
I actually don't want to use ignore because of confusion with the git ignore, because this is not git ignore.

Reply:
done